portfolio/pom.xml

```diff
@@ -34,8 +34,7 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>jimmycolema-step-2020</deploy.projectId>
```

Comment:
This is fine now but for future reference it should probably be in a different branch, it'd not really part of "Adding Stylistic Features to Portfolio"

Reply:
Point taken